FluentScreenRecorder/MultilingualResources/FluentScreenRecorder.it.xlf

```diff
@@ -60,7 +60,7 @@
         </trans-unit>
         <trans-unit id="GitHub" translate="yes" xml:space="preserve">
           <source>View GitHub repository</source>
-          <target state="translated">Vedi il repository su GitHub</target>
+          <target state="translated">Vedi la repository su GitHub</target>
```

Comment:
In Inglese, repository è maschile, perciò anche se a me suona meglio è scorretto

Reply:
Direi che usare semplicemente "Repository su GitHub" possa risolvere tutti i dubbi a riguardo.